README.md

```diff
@@ -56,21 +56,21 @@ export default class App extends Component {
   <tr>
     <td>numInputs</td>
     <td>number</td>
-    <td><strong>true</strong></td>
+    <td>true</td>
```

Comment:
I think this was intentional. Can we keep the strong tags in place. I'll merge the PR right away!

Reply:
HI @apollonian it makes sense to remove the strong. Please state the reason why they could have been emphasized. I shall open a new issue regarding this, fix it by keeping the strong tags in place and make a pr again.

Reply:
Please look for the pr